kong-ngx-build

```diff
@@ -411,24 +411,28 @@ parse_version() {
   [[ -z $1 ]] && fatal 'missing arg $1 when invoking parse_version()'
   [[ -z $2 ]] && fatal 'missing arg $2 when invoking parse_version()'
 
+  local ver
   local subj=$1
-  local -n ver=$2
   local re='^(-rc[0-9]+$)?[^0-9]*([0-9a-zA-Z]*)(.*?)$'
```

Comment:
We could always do a 

```
  [[ ${BASH_VERSION} =~ 3.* ]] && fatal "please upgrade to bash 4 at least"
```

but then we would have gone full circle :D

Reply:
I just added a commit testing the parsing utilities in Bash 3/4/5.